appwrite/website
# Add Database Backups announcement

## What does this PR do?

- Adds database backups announcement
- Adds entry to changelog
- Add blog post assets
- Fixes syntax error with deno changelog entry

## Test Plan

- Run the website and navigate to `/blog/post/introducing-database-backups`
- Confirm that blog post exists and that the assets are displayed correctly
- Navigate to `/changelog` and confirm that new database backups entry is displayed
- Confirm that deno announcement changelog entry is displayed

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes
